src/views/v2/Bridge/AssetPicker/TokenItem.tsx

```diff
@@ -153,16 +154,25 @@ function TokenItem(props: TokenItemProps) {
           {props.isFetchingBalance && props.balance === null ? (
```

Comment:
I think this is definitely something we can extract out into its own components `<TokenPrice>` and `<TokenBalance>`. With nested ?:?: hard to understand whats really happening.

But we can leave it as-is in the v2 folder, just do it for v3